operator/params.yaml

```diff
@@ -22,6 +22,22 @@ NODE_MEM_LIMIT_MIB:
   description: "Memory limit (in MiB) for the Cassandra node pods."
   default: 4096
 
+METRICS_CPU_MC:
```

Comment:
@mpereira Okay i will prefix it as suggested , and for time being its okay. but I used METRICS for  general purpose as Kudo Cassandra is exporting metrics which could be used for other tools, and if customer wants to use Prometheus we are specifically running this command for it  `kubectl create -f resources/service-monitor.yaml` , thats why `service-monitor.yaml` was not included in templating (hence cant provide param to service-monitor.yaml).

cc: @samvantran

Reply:
Regarding requiring a manual `kubectl create -f resources/service-monitor.yaml` command, can we just enclose the whole `resources/service-monitor.yaml` in an "if prometheus exporter enabled" block instead?

Reply:
@mpereira @samvantran 
Although i have already changed Param to prefix with PROMETHEUS_EXPORTER, i just want to confirm again that should we be targeting only Prometheus ? or just make it more general and ask customer to run command manually to see metrics on Prometheus. 
if we are targeting Prometheus only then this service-monitor object can be automated can be automated.Currently  `resources/service-monitor.yaml` is not  included in operator hence its manual process and at the same time we cant use any param in that. 

also just FYI General approach is followed by kudo kafka.

Reply:
Yes, let's focus on Prometheus.

Reply:
I think for beta, it's fine for us to specifically target prometheus since we're also using the criteo/prom-exporter. But when we GA, if we decide to output custom/native metrics then we can generalize some of these configs